jasmine/spec/inverted-index-spec.js

```diff
@@ -0,0 +1,79 @@
+// Instanciate new Index
+var index = new Index();
+
+// Run createIndex method and make sure it completes before any test is run
+beforeEach(function(done) {
+    index.createIndex('/jasmine/spec/test.json').then(function() {
```

Comment:
**Code Styling - Nitpick**

AirBnB suggests the use of 2 spaces for indentation. Refer [here] (https://github.com/airbnb/javascript#whitespace--spaces)